tekton/src/pipelines/fvt-data-dictionary.yml.j2

```diff
@@ -26,3 +26,15 @@ spec:
     # 1. ADD FVT
     # -------------------------------------------------------------------------
     {{ lookup('template', 'taskdefs/fvt-data-dictionary/data-dictionary.yml.j2') | indent(4) }}
+    - name: imagescan
```

Comment:
I don't think we can add this for data dictionary yet .. it's not a real MAS app, would leave it out for now ... get the apps done first.  this will need special handling